apps/docs/src/pages/components/header.mdx

```diff
@@ -0,0 +1,279 @@
+import { Callout } from "nextra/components";
+import { headerClasses, HvHeader } from "@hitachivantara/uikit-react-core";
+
+import Playground from "@docs/components/code/Playground";
+import { Description } from "@docs/components/page/Description";
+import { Page } from "@docs/components/page/Page";
+import { getComponentData } from "@docs/utils/component";
+
+export const getStaticProps = async ({ params }) => {
+  const meta = await getComponentData("Header", "core", headerClasses, [
+    "Actions",
+    "Brand",
+    "Navigation",
+  ]);
+  return { props: { ssg: { meta } } };
+};
+
+<Description />
+
+<Page>
+
+### Usage
+
+The example below shows a simple header with a brand, navigation, and actions. If you resize the window to a smaller size, the navigation will be hidden and a menu button will appear.
```

Comment:
Can we have a simpler sample without Navigation? It adds quite a bit of complexity, and Pentaho+ Header designs don't seem to use/promote it anymore anyways.

Also, can we update the docs to better explain what `HvHeader` **is**? I think the key points are that it's composable (user can add any pieces), but we provide `HvHeaderBrand`, `HvHeaderNavigation`, `HvHeaderActions`.